src/lib/InteractionManager.ts

```diff
@@ -138,8 +138,9 @@ export class InteractionManager {
           this._longestInteractionMap.delete(interaction.id);
         }
       }
-    }
 
-    this._onAfterProcessingInteraction?.(interaction!);
+      // Call any post-processing on the interaction
+      this._onAfterProcessingINPCandidate?.(interaction!);
```

Comment:
```suggestion
      this._onAfterProcessingINPCandidate?.(interaction);
```

Turns out I shouldn't have ignored TypeScripts warnings on this!